src/index.ts

```diff
@@ -99,9 +106,19 @@ async function serverChannel(
 
   // eslint-disable-next-line react-hooks/rules-of-hooks
   const gitInfoState = useAddonState<GitInfoPayload>(channel, GIT_INFO);
-  observeGitInfo(5000, (info) => {
-    gitInfoState.value = info;
-  });
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const gitInfoError = useAddonState<Error>(channel, GIT_INFO_ERROR);
+
+  observeGitInfo(
+    5000,
+    (info) => {
+      gitInfoState.value = info;
+    },
+    (error: Error) => {
+      gitInfoError.value = error;
+    }
+  );
```

Comment:
Theoretically, it's possible there's some other issue with git (like it gets corrupted or something). I'm not sure we really need to cope with that, but just flagging it here (a git failure doesn't necessarily mean git isn't setup).

Reply:
That makes sense. In this case, whatever the error occurring would otherwise fail to start the cli, so I think we'd want to handle it similarly to how we do. I think the stacktrace will be useful if users encounter other types of errors that cause this.

Reply:
Nevermind. This was another issue with my code.